src/models/oryp10/external-overview.md

```diff
@@ -0,0 +1,64 @@
+# Oryx Pro (External Overview)
+
+## Left side overview:
+
+![Left Ports](./img/ports-left.webp)
+
+## Right side overview:
+
+![Right Ports](./img/ports-right.webp)
+
+## Front side overview:
+
+![Front Ports](./img/ports-front.webp)
+
+## Back side overview:
+
+![Back Ports](./img/ports-back.webp)
+
+## Keyboard & touchpad overview:
+
+![Keyboard & Touchpad](./img/keyboard-touchpad.webp)
+
+## Materials:
+
+|Part                              |Material                |
+|----------------------------------|------------------------|
+|LCD back cover (lid)              |Aluminum                |
+|LCD front cover (bezel)           |Plastic                 |
+|Top case (palm rests/port covers) |Aluminum                |
```

Comment:
The top case is plastic on oryp10 and oryp9. I think I missed this on the oryp9 docs.

Reply:
Yikes. Fixed with 8e530b15676b8b50667f5f6813702e0d199dc4af.